src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationService.java

```diff
@@ -30,10 +43,24 @@ public void processServiceBusMessage(ErrorMsg message) {
             message.errorDescription,
             message.id
         );
+        ErrorNotification entity = new ErrorNotification(
+            entityManager.getReference(ProcessEvent.class, message.eventId),
+            message.errorCode.name(),
+            message.documentControlNumber,
+            message.id
+        );
+
+        try {
+            ErrorNotificationResponse response = client.notify(request);
+
+            entity.setNotificationId(response.getNotificationId());
```

Comment:
How about adding notification id to the constructor? Looks like we can just create a proper instance in one go.